scripts/clear-all-fault-leds.sh

```diff
@@ -80,7 +80,7 @@ else
         GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
         | grep -Ev "$excluded_groups" | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
-        echo "$line" | grep "unit\|core" >/dev/null
+        echo "$line" | grep "powersupply\|dimm\|cpu\|fan" >/dev/null
```

Comment:
use dcm instead of cpu

Reply:
is this path working ?

Reply:
could you paste an echo command in above and here and try and see if both works fine or not

Reply:
Okay will add dcm, and check with echo

Reply:
I tried adding echo, but this[2nd] flow is excluding everything so changed it to exclude only the required one. Thanks .